.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+### Requirements
+* Read the [HyperSpy developer guide](https://hyperspy.org/hyperspy-doc/current/dev_guide/intro.html).
```

Comment:
```suggestion
* Until `RosettaSciIO` has a contributor guide, you can read the [HyperSpy developer guide](https://hyperspy.org/hyperspy-doc/current/dev_guide/intro.html) to familiarise yourself with how to contribute.
```
Once there is a contributor guide in the `RosettaSciIO` docs (possibly with reference to the hyperspy developer guide and/ or other), we should update this link, but in the meantime, this is useful as a reference!